.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.formatting.provider": "autopep8"
+}
```

Comment:
This is probably better excluded since we don't generally require developers to use anything though we do apply black and isort here and there.